src/uav_swarm_sim/planning/lloyd_partition.py

```
@@ -569,3 +569,215 @@ class LloydCvtDecomposer(_LloydDecomposer):
+        from .energy_balance import estimate_fast_from_area
+
+        empty = estimate_fast_from_area(
+            self._ctx, self._states[i], alt=self._alt, area_m2=0.0,
+            centroid_xy=(0.0, 0.0), fallback_pose=self._fallbacks[i],
+        )
+        return float(empty.budget_j - empty.demand_j)
+
+    def initial(self, n: int) -> np.ndarray:
+        return np.zeros(n, dtype=float)
+
+    def excluded(self, n: int) -> np.ndarray:
```

Ruff (0.16.3)

[warning] Unused method argument: `n`

(ARG002)